CONTRIBUTING.md

```diff
@@ -55,9 +55,16 @@ conda install pytorch -c pytorch-nightly
 pip install cmake ninja
 ```
 
+### Install other requirements
+
+```bash
+pip install -r requirements.txt
```

Comment:
We do not actively maintain `requirements.txt`. (which is not good, but at the same time I always wonder the value of `requirements.txt`, especially there is similar section in `setup.py`) 

We have better description in line 85. Can we instead update it?
`librosa`, `requests`, `pyyaml`, are missing from there.
Other missing ones are `fairseq` and `transformers`, but they are very purpose-specific, so we can omit.

Reply:
Sure, I'll update them.

Reply:
I think we also need `soundfile` otherwise many tests will fail. Should we also add it to line 85?

Reply:
Which ones are failing? When a dependency is missing our tests are supposed to be skipping them.

Reply:
Something like this one
```FAILED torchaudio_unittest/backend/soundfile/save_test.py::MockedSaveTest::test_sph_sph_int16_16000_1_True_ALAW_32 - ModuleNotFoundError: No module named 'soundfile'```
also some in this file ```https://github.com/pytorch/audio/blob/master/test/torchaudio_unittest/kaldi_io_test.py```

Reply:
Ok, these are bugs, I will fix them. So, can you update the description so that all of them are optional?
`librosa`, `requests`, `soundfile`, `kaldi_io`, `transformers` and `fariseq`.

Reply:
Thanks for pointing it out. I've opened an issue (#1589) regarding this.